src/pages/Admin/Charity/Settings/Form.tsx

```diff
@@ -71,6 +65,20 @@ export default function Form(props: Props) {
         </p>
       </div>
 
+      <div>
+        <CheckField<FV>
+          name="programDonateDisabled"
+          classes={{ label: "font-medium" }}
+        >
+          Disable Program-based donations
+        </CheckField>
+        <p className="text-xs sm:text-sm text-navy-l1 italic mt-1">
+          Program based donations are allowed by default, enabling donors to
```

Comment:
My bad! The original copy had a grammar error. 
```suggestion
          Program-based donations are allowed by default, enabling donors to
```